components/SubmissionComments.test.js

```diff
@@ -160,28 +160,26 @@ describe('Test SubmissionComments Component', () => {
 
     const editButton = container.querySelector('.btn-outline-info.btn-sm')
     fireEvent.click(editButton)
+    const discardButton = container.querySelector('.btn-light')
 
-    await waitFor(() => {
-      const discardButton = container.querySelector('.btn-light')
-      expect(discardButton).toBeInTheDocument()
-      expect(container.querySelector('.btn-info')).toBeInTheDocument()
+    expect(discardButton).toBeInTheDocument()
+    expect(container.querySelector('.btn-info')).toBeInTheDocument()
 
-      const editButtonTwo = container.querySelector('.btn-outline-info.btn-sm')
-      fireEvent.click(editButtonTwo)
+    const editButtonTwo = container.querySelector('.btn-outline-info.btn-sm')
+    fireEvent.click(editButtonTwo)
 
-      const warningText =
+    expect(
+      await findByText(
         'You can only edit one comment in a single comment chain at a time.'
+      )
+    ).toBeTruthy()
 
-      waitFor(() => {
-        expect(findByText(warningText)).toBeTruthy()
-      })
+    fireEvent.click(discardButton)
```

Comment:
FYI, you'll notice that `fireEvent` is used in some files in the codebase, but `userEvent` is preferred because it's closer to how a user actually interacts with an element.